### PR TITLE
modules: add elasticsearch

### PR DESCRIPTION
**Pull request by ilyam8**

### CODE REVIEW

File: modules/elasticsearch/charts.go [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+			Fam:   "indices indexing",
+			Ctx:   "elasticsearch.node_indices_indexing",
+			Dims: Dims{
+				{ID: "node_indices_indexing_index_total", Name: "index", Algo: module.Incremental},
+			},
+		},
+		{
+			ID:    "node_indices_indexing_operations_current",
+			Title: "Indexing Operations Current",
+			Units: "operations",
```
Comment (prologic) [inline modules/elasticsearch/charts.go:32]: Is this missing an `/s`?

File: modules/elasticsearch/charts.go [hunk 2]
```diff
@@ -0,0 +1,581 @@
...
+			Type:  module.Stacked,
+			Dims: Dims{
+				{ID: "node_indices_search_query_total", Name: "queries", Algo: module.Incremental},
+				{ID: "node_indices_search_fetch_total", Name: "fetches", Algo: module.Incremental},
+			},
+		},
+		{
+			ID:    "node_indices_search_operations_current",
+			Title: "Search Operations Current",
+			Units: "operations",
```
Comment (prologic) [inline modules/elasticsearch/charts.go:65]: Same here?

File: modules/elasticsearch/charts.go [hunk 3]
```diff
@@ -0,0 +1,581 @@
...
+				{ID: "node_indices_segments_index_writer_memory_in_bytes", Name: "index_writer"},
+				{ID: "node_indices_segments_version_map_memory_in_bytes", Name: "version_map"},
+				{ID: "node_indices_segments_fixed_bit_set_memory_in_bytes", Name: "fixed_bit_set"},
+			},
+		},
+		// Indices Translog
+		{
+			ID:    "node_indices_translog_operations",
+			Title: "Translog Operations",
+			Units: "operations",
```
Comment (prologic) [inline modules/elasticsearch/charts.go:194]: And here?

**Review by prologic — APPROVED** [review]

File: modules/elasticsearch/charts.go (hunk 1 above)
Comment (ilyam8) [reply modules/elasticsearch/charts.go:32]: i add `/s` to rates which are dimensions with `incremental` algorithm, this one is not

File: modules/elasticsearch/charts.go (hunk 2 above)
Comment (ilyam8) [reply modules/elasticsearch/charts.go:65]: not `incremental`

File: modules/elasticsearch/charts.go (hunk 3 above)
Comment (ilyam8) [reply modules/elasticsearch/charts.go:194]: i believe this one is not `incremental` too

**Pull request merged**